python/ray/ml/predictors/integrations/tensorflow/tensorflow_predictor.py

```diff
@@ -136,9 +136,14 @@ def build_model(self):
                 data = data[feature_columns]
             data = data.values
         else:
-            data = data[:, feature_columns]
+            if feature_columns:
+                data = data[:, feature_columns]
 
-        tensor = tf.convert_to_tensor(data, dtype=dtype)
+        row, column = data.shape
+        if column == 1 and row > 0 and isinstance(data[0][0], tf.Tensor):
+            tensor = tf.stack(data[:, 0].tolist())
+        else:
+            tensor = tf.convert_to_tensor(data, dtype=dtype)
```

Comment:
@clarkzinzow if I comment out line 134 - 146 and replace it with
```
data = data["value"]
tensor = tf.convert_to_tensor(data.to_numpy())
``` 
Somehow it still throws error:
```
ValueError: Failed to convert a NumPy array to a Tensor (Unsupported object type numpy.ndarray).
```